app/graphql/mutations/import_gtfs_feeds.rb

```diff
@@ -0,0 +1,27 @@
+module Mutations
+  class ImportGtfsFeeds < BaseMutation
+    argument :dataprovider_id, ID, required: false
+    argument :feed, String, required: false
+
+    type Types::StatusType
+
+    def resolve(dataprovider_id:, feed:)
+      status, status_code = ["Import failed: No data provider found", 404] unless dataprovider_id.present?
```

Comment:
do we want to take care of line lengths?

Reply:
we could think of removing this line in favor of requiring the arguments

```
argument :data_provider_id, ID, required: true
argument :feed, String, required: true
```

not passing these two makes no sense for the mutation.